package.json

```diff
@@ -85,6 +85,7 @@
       "yarn lint:js"
     ],
     "src/**/*.scss": [
+      "yarn lint:style",
```

Comment:
Doesn't `lint:style` run `stylelint`?

Reply:
https://github.com/yankaindustries/mc-components/blob/develop/package.json#L31

It does indeed!